Networking/Networking/Network/NetworkError.swift

```diff
@@ -21,6 +21,19 @@ public enum NetworkError: Error, Equatable {
 
     /// Error for REST API requests with invalid cookie nonce
     case invalidCookieNonce
+
+    var response: Data? {
```

Comment:
Kindly consider adding a comment to explain this property.

Reply:
Also done in d2de940cb552fb2102c003ef68d052e69d15cfaa.